src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure implements ObtainArea {
```

Comment:
```suggestion
public class IsoscelesTrapezoid extends Figure {
```

Давай добавим этому классу поля, которые характеригуют равнобедренную трапецию и используем их для подсчета площади фигуры (то же самое для остальных классов фигур)